src/sdk/ActivityReceivedEventArgs.ts

```diff
@@ -4,8 +4,12 @@
 /* eslint-disable max-classes-per-file */
 import { PullAudioOutputStream } from "./Audio/AudioOutputStream";
 
-export interface IActivity {
+interface IActivity {
+    conversation?: any;
+    id?: string;
+    replyToId?: string;
     type?: string;
+    text?: string;
     speak?: any;
```

Comment:
I'm wondering if you also would like to add "name", "value" and "inputHint" to IActivity. But since the return type has changed to any, it should not block client from getting them.